README.md

```diff
@@ -16,36 +16,26 @@ You can also setup webhooks via Topgg.Webhook, look down below at the examples f
 
 # Links
 
-[Documentation](https://topggjs.rtfd.io)
+[Documentation](https://topgg.js.org)
```

Comment:
has this already been set up?

Reply:
> has this already been set up?

we have to have a working app before I can make a request for js.org, though I don't think it's bad to have it here since it'll only be released to npm once we have that up. I'll only make the release after that, so it'll just sit in GitHub for a while.